File: app/src/main/java/temple/edu/zomato_randomizer/MainActivity.java
package temple.edu.zomato_randomizer;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    boolean isSavedLogin;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (isSavedLogin){
            //  auto load options activity
        }

        setContentView(R.layout.activity_login);


    }
}

File: app/src/main/java/temple/edu/random/activities/MainActivity.kt
package temple.edu.random.activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import temple.edu.random.R

class MainActivity : AppCompatActivity() {
    var isSavedLogin = false
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        if (isSavedLogin) {
            //  auto load options activity
        }
        setContentView(R.layout.activity_login)
    }
}